lib/portage/dbapi/bintree.py

```diff
@@ -1234,7 +1234,16 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER")
+        ret = os.system(portage_trust_helper)
+        if ret != 0:
+            raise OSError(_("Failed to run trust helper for binary package verification"))
+        return
+
     def _populate_remote(self, getbinpkg_refresh=True):
+        self._run_trust_helper()
```

Comment:
Let's conditionalise this on `FEATURES="binpkg-request-signature"` (maybe `FEATURES="binpkg-request-signature"` or `FEATURES="binpkg-signing"`, but I don't see that much point there)?

If you agree with that, we can shove it under the `if` on line 1250.

Reply:
Bleh, okay. Let's put it under that `if` I mentioned, but put a comment there. I think we should circle back and change that behaviour you mention at some point, not sure.

Reply:
OK done.